R/imputation.R

```diff
@@ -0,0 +1,326 @@
+##' @title Quantitative mass spectrometry data imputation
```

Comment:
very minor point: so far we used `#'` for roxygen comments - just for consistency reasons I think it might be better to replace these `##'` with `#'`.